### PR TITLE
adds sanity portfolios

### PR DESCRIPTION
**Pull request by joelhooks**
this is a quick draft of modeling a catalog of quality portfolios in Sanity

did a little tour: https://www.loom.com/share/31accbb17f4341d8adcd820d91885503

image

image

### CODE REVIEW

File: src/pages/examples-of-good-developer-portfolios/[slug].tsx [hunk 1]
```diff
@@ -0,0 +1,137 @@
+import * as React from 'react'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+import mdxComponents from 'components/mdx'
+import renderToString from 'next-mdx-remote/render-to-string'
+import Image from 'next/image'
+import {NextSeo} from 'next-seo'
+import {useRouter} from 'next/router'
+
+const Portfolio = (props: any) => {
```
Comment (joelhooks) [inline src/pages/examples-of-good-developer-portfolios/[slug].tsx:10]: this is a cut and paste from the blog page, didn't spend much time in here

File: src/pages/examples-of-good-developer-portfolios/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import * as React from 'react'
+import groq from 'groq'
+import {sanityClient} from 'utils/sanity-client'
+import Image from 'next/image'
+import {useRouter} from 'next/router'
+import Link from 'next/link'
+
+const Portfolio: React.FC<{portfolios: any}> = (props) => {
```
Comment (joelhooks) [inline src/pages/examples-of-good-developer-portfolios/index.tsx:8]: super quick index just to get it displaying

File: studio/schemas/documents/resource.js [hunk 3]
```diff
@@ -108,6 +108,10 @@ export default {
             title: 'talk',
             value: 'talk',
           },
+          {
+            title: 'portfolio',
+            value: 'portfolio',
```
Comment (joelhooks) [inline studio/schemas/documents/resource.js:113]: added a `portfolio` type

File: studio/src/structure/portfolio.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import S from '@sanity/desk-tool/structure-builder'
+
+const portfolio = S.listItem().title('Portfolio').child(
```
Comment (joelhooks) [inline studio/src/structure/portfolio.js:3]: this is a very simple "desk" item that adds the filtered resource list to the main menu in Sanity Studio

**Comment by MaggieAppleton** [comment]
Nice - I poked around to figure out how sanity works. Added a few more portfolios to the database, uploaded more image previews to cloudinary, and played around with the layout a bit.

All still rough / WIP but has potential to be a good resource

image

File: src/pages/examples-of-good-developer-portfolios/index.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+  const router = useRouter()
+  const url = process.env.NEXT_PUBLIC_DEPLOYMENT_URL + router.asPath
+
+  return (
+    <>
+      <section className="mx-auto max-w-screen-lg lg:mt-14 md:mt-8 mt-3 mb-16">
+        <h1 className="max-w-screen-md lg:text-6xl md:text-5xl sm:text-4xl text-4xl font-bold leading-tighter">
+          Gold Star Developer Portfolios
+        </h1>
+        <section className="grid grid-cols-2 gap-x-8 lg:gap-x-12 mt-12 gap-y-6 lg:gap-y-8">
```
Comment (vojtaholik) [inline src/pages/examples-of-good-developer-portfolios/index.tsx:19]:
```suggestion
        <section className="grid sm:grid-cols-2 grid-cols-1 sm:gap-8 gap-5 mt-12">
```

**Review by vojtaholik — APPROVED** [review]

**Pull request merged**